src/main/java/core/basesyntax/figuresuppliers/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.figuresuppliers;
+
+import core.basesyntax.color.ColorSupplier;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        int sizeOne = new Random().nextInt(10) + 1;
+        int sizeTwo = new Random().nextInt(10) + 1;
+        int sizeThree = new Random().nextInt(10) + 1;
+
+        int index = new Random().nextInt(FigureType.values().length);

```

Comment:
I would recommend to make `Random` instance a class member. It brings some benefits, like we do not need to create instance each time method is invoked.